### ISSUE TITLE
Spot Checking Balloon interactions 

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#419]
As discussed with @jessegreenberg, I am just spot checking balloon interactions before RC Testing:
- [x] dragging with 1 balloon, 
- [x] dragging with 2 balloons, 
- [x] jumping with 1 balloon, 
- [x] jumping with 2 balloons, 
- [x] independent movement with 1 balloon,
- [x] independent movement with 2 balloons.

**Comment by terracoda** [comment]
When dragging up and down "Very close to sweater" I do not hear any progress up dates. I get black steps in between directional changes and "At top." and "At bottom."

I think this is a side effect of the fact that for "Near Sweater" or "Near Wall" we do not get progress updates.

In the case of Near Sweater, we always hear, "Near Sweater". In the case of "Near Wall" and "Very Close to wall" we have the benefit of changes between "wall, upper wall and lower wall."

@jessegreenberg, can vertical dragging alerts in the "Very close to sweater" column work the same as the "Near sweater" column? So, instead of blank steps, we hear "Very close to sweater."

**Comment by terracoda** [comment]
When two balloons are present, I seem to hear "No change in charges" right after I pick up charge on the sweater.

It's almost like there is some extra sensitivity to that alert when two balloons are present.

Moving to new issue.

**Comment by terracoda** [comment]
Done spot checking.
